assets/js/common/CheckableWarningMessage/CheckableWarningMessage.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Input from '@common/Input';
+import { noop } from 'lodash';
+import { EOS_WARNING_OUTLINED } from 'eos-icons-react';
+
+function CheckableWarningMessage({
+  hideCheckbox = false,
+  warningText,
+  checked,
+  setChecked = noop,
```

Comment:
Maybe better `onChecked`?
And in `onChange` you simply do `onChange={onChecked}`.
It should be the user of this module responsabilite to decide what to do them